object/search.go

```diff
@@ -37,6 +41,14 @@ func (m SearchMatchType) ToV2() v2object.MatchType {
 		return v2object.MatchNotPresent
 	case MatchCommonPrefix:
 		return v2object.MatchCommonPrefix
+	case MatchNumGT:
+		return 5
```

Comment:
looks extremely strange to me but since you are _that_ against api-go, i cannot say anything about it

Reply:
This line sucks, really.

Reply:
i can add one more PR if u need

Reply:
at least constants from `.pb` file. can be merged in the https://github.com/nspcc-dev/neofs-api-go/pull/437. or we can add the same `v2object` constants, not a big work

Reply:
done